packages/x-components/package.json

```diff
@@ -67,9 +67,9 @@
     "@empathyco/x-types": "^10.0.0-alpha.13",
     "@types/resize-observer-browser": "~0.1.5",
     "magic-string": "~0.25.7",
-    "nanoid": "^3.1.30",
+    "nanoid": "~3.1.30",
```

Comment:
Non empathy dependencies should only match patch version.

Reply:
then we should do the same for:

- `"vue": "^2.6.0"`
- `"vuex": "^3.0.0"`
- `"@vue/vue2-jest": "^27.0.0-alpha.3",`
- `"vue-template-compiler": "^2.6.10"`

shouldn't we?

Reply:
Yes and no.

- Vue and Vuex should be "peer dependencies". We wan't them to be more flexible, and to use the user project version in any case, that's why we specified them with the minor instead of the patch one. We would probably have to re-check Vuex minimum version, as now the installed one is 3.6, and I'm not sure if all the features we are using are available since the 3.0.0.
- Vue template compiler should match Vue version, I would probably change it to `^2.6.0`. Now they end up being the same version, but if we accidentaly install Vue 2.6.1 for example, I think components will fail at compile time or at least throw some errors or warnings.
- Vue2-jest should be changed. I'll change it also in this PR, good catch.

Reply:
I've just checked [vue2-jest](https://github.com/vuejs/vue-jest/commit/6edfa9cd93be05ca4c9c41849db668dbf2ae92d5), and they are treating alphas as release candidates 🤣,  so we should be safer than I thought.